modules/programs/lutris.nix

```diff
@@ -0,0 +1,152 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.programs.lutris;
+
+  cemuConf = ''
+    cemu:
```

Comment:
Doing yaml configurations in this way will prevent users of the module from doing settings that are not provided in the module. Are you sure that it won't be necessary to allow users deeper access?

Reply:
There will be a need to allow users deeper access to these. I was thinking of adding an extraConf part for cemu, dolphin, duckstation and ppsspp that is user modfiable. If there is a better method for this, please let me know. Also, any example is greatly appreciated!